src/components/Calculator/CalculatorBtn.js

```diff
@@ -1,18 +1,14 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class CalculatorBtn extends React.Component {
-  render() {
-    const { action, showName, name } = this.props;
-    return (
-      <div className={`calculator-btn action-${name}`}>
-        <button type="button" className="calculator-showName" onClick={action}>
-          {showName ?? name}
-        </button>
-      </div>
-    );
-  }
+function CalculatorBtn({ action, showName, name }) {
```

Comment:
- [ ] Good job so far by refactoring your code :clap:  But Please use the arrow function instead of the regular function for the reason I mentioned above :arrow_up: